sdk/android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         javaVersion = JavaVersion.VERSION_1_8
 
         // App dependencies
-        exoplayerVersion = '2.9.5'
```

Comment:
Shall we use the latest version?

Reply:
we forced to use 2.9.3 check release notes